src/views/partials/main/footer.hbs

```diff
@@ -40,6 +40,12 @@
               <span class="fab fa-instagram"></span>
             </a>
           </li>
+          <li class="nav-item">
+            <a href="https://www.meetup.com/lernwerkstatt/" aria-label="Unsere Meetup Seite"
```

Comment:
suggestion: maybe we should have `noopener noreferrer` here? What do you think @jarjan ?

Reply:
@pheeria I think we can leave as is.